sdks/open-format/src/core/contract.ts

```diff
@@ -420,6 +415,9 @@ export async function burn({
   return receipt;
 }
 
+const defaultNFTStorageToken =
+  'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJkaWQ6ZXRocjoweDVkMjJDZDg2M2Y2NUVBN0ZjZjI3MEE5MUY2NTE5Nzc4OGRhRjU4NmMiLCJpc3MiOiJuZnQtc3RvcmFnZSIsImlhdCI6MTY1OTAxMTU3OTQ2NSwibmFtZSI6Ik9wZW4gRm9ybWF0In0.v-0FkXH0IpVB_JdyG6Ho6kXPqy4DOVr_HZzsks3DES4';
+
```

Comment:
I think we want to remove the default token entirely, so we're not exposing our own token. Basically if they don't provide a token, we don't do the upload and they need to provide their own IPFS URL.